assets/js/live_vue/utils.ts

```diff
@@ -33,9 +33,10 @@ export const flatMapKeys = <T>(
  * @returns The component if found, otherwise throws an error with a list of available components.
  */
 export const findComponent = (components: ComponentMap, name: string): ComponentOrComponentPromise => {
-  // we're looking for a component by name or path suffix.
+  // we're looking for a component by exact filename match
   for (const [key, value] of Object.entries(components)) {
-    if (key.endsWith(`${name}.vue`) || key.endsWith(`${name}/index.vue`)) {
+    const fileName = key.split('/').pop() // Get the actual filename
```

Comment:
I understand why you've done it, but I can explain the reason for the previous behaviour. 

Imagine you have two components, `components/workspaces/List.vue` and `conponents/accounts/List.vue`. 

With my previous approach, you could type either full path or just last unambigious part. With yours I think it's no longer possible. 

But! 

I think you can adjust it. Something along the lines to split both candidate and searched component by `/` and try to match starting from last index. 

Would be great if you (or AMP) could do it ;)